operations/configuration_management/CFS_Key_Management.md

```diff
@@ -0,0 +1,33 @@
+## CFS Key Management and Permission Denied Errors
+
+Configuration Framework Service \(CFS\) manages it's own keys separate from keys for communication between CFS and the components or images that it is configuring.  These are separate from the keys used by users and should not need to be managed.
+
+### Permission Denied Errors
+
+If Ansible is unable to connect with it's target and fails with an `Unreachable - Permission denied` error the first place to check is the `cfs-state-reporter` on the target node.
```

Comment:
- it's -> its
- Add a comma after error in the second sentence

Reply:
fixed